intg/src/main/java/org/apache/atlas/model/glossary/relations/AtlasRelatedTermHeader.java

```diff
@@ -32,6 +35,27 @@
     private String steward;
     private String source;
 
+    public Map<String, Object> getOtherAttributes() {
+        return otherAttributes;
+    }
+
+    private Map<String, Object> otherAttributes;
+
+
+
+    public void setOtherAttributes(Map<String, List<String>> rs, List includeAttributes) {
+        if (this.otherAttributes == null) {
+            this.otherAttributes = new HashMap<>();
+        }
+        if (includeAttributes!=null) {
+            for (String key : rs.keySet()) {
```

Comment:
Suggestion: I think iterating over includeAttributes will be better in some cases e.g. in case it has less elements iterations will be less
Not necessary